### ISSUE TITLE
Keener - invoices not being emailed

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1199]
In GitLab by @kdjstudios on Oct 17, 2018, 10:31

**Submitted by:** Gaylan Garrett <gaylan@keenercom.net>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-10-17-75213/conversation

**Server:** External

**Client/Site:** Keener

**Account:** Multiple

**Issue:** 

I have just been working with a client 3561 and 3574 because they have not gotten emails of their invoices since May. I emailed them to her again waiting for the blue bar at the top to say it was emailed before emailing the next one. She responded back and said she only got received of the 10 invoices.

 

So, now I have gone into the account and sent 1 email and then go out of the account and then go back into the account and send the next invoice and do this process until I have emailed all 5 invoices for each of the two accounts.

 

She has not responded back yet as to what she has received. I have had multiple clients say they are not receiving their invoices even though my report says it was emailed successfully and they have checked spam and junk folders and still do not have it.

 

Thanks for looking into this issue for me.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 17, 2018, 10:51

@smasih @amishra @pchaudhary @tim.traylor 

I looked at both example accounts provided and they both have the same email. "chanera.walker@pdicorp.com". Judging from the last email issues Gaylan was having (#1141 and #1147) I also checked and their file attachment does appear to be working. I can download the file without issue. I looked further back and Gaylan brought up other client accounts that are not receiving the email invoices in (#1052); which there has been no progress made on. EDIT: After looking farther back it would appear that #1052 is just Gaylan reopening the #566 ticket. 

Judging from reading the past tickets that other clients are received the emails fine and the simple fact that Gaylan can send direct emails to these client. It would tend to appear that we are sending the emails out, but they are being blocked by the client. Yet Gaylan is stating that the client does not see these emails in their spam or junk, so I am at a loss on where these emails are being blocked at; if that is the case. 

In Summary, we first need to confirm that all the emails are being sent out from our system and provide proof of such.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 17, 2018, 14:54

Gaylan wrote:

> I have another client to add to the list that is not getting the email although the report says it was sent. Account 1273 and 1295.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 18, 2018, 08:13

@smasih @pchaudhary @amishra @tim.traylor 

Gaylan wrote:

> I have confirmation from 3561 and 3574 that they are not receiving the email of the invoice even though I get the snip it at the top when I send it that says "  invoice # ----- was successfully emailed "  I have gotten that on each of the invoices that I have attempted to email to all 4 ( 1295 and 1273 also) clients. 
> 
> Just wanted to keep you updated.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Oct 23, 2018, 08:49

@kdjstudios Kyle, the email is being sent through our servers, so Robert or Ben will need to trace the logs and confirm if it was sent.  If we see it was sent we can provide the logs to the client for them to address with their email provider or tech support.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 23, 2018, 09:15

@tim.traylor 

Thanks, I have requested their aid in CERB and will follow up accordingly.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 13, 2018, 14:37

closed
